misk-hibernate-testing/src/main/kotlin/misk/jdbc/VitessScaleSafetyChecks.kt

```diff
@@ -169,7 +170,7 @@ class VitessScaleSafetyChecks(
 
   private val fullScatterDetector = FullScatterDetector()
   private val crossEntityGroupTransactionDetector = CowriteDetector()
-  val enabled = ThreadLocal.withInitial { true }
+  val enabled: ThreadLocal<Boolean> = ThreadLocal.withInitial { true }
```

Comment:
IntelliJ recommends explicit types when the implicit type comes from Java.

Reply:
sure